eng/testing/linker/SupportFiles/Directory.Build.targets

```diff
@@ -48,4 +62,9 @@
       TargetFramework="$(TargetFramework)" />
   </ItemGroup>
 
+  <PropertyGroup>
+    <BundleDir>$([MSBuild]::NormalizeDirectory('$(MSBuildProjectDirectory)', '$(OutputPath)', 'AppBundle'))</BundleDir>
+    <WasmMainAssemblyFileName>project.dll</WasmMainAssemblyFileName>
```

Comment:
Shouldn't this be defaulted to the `$(AssemblyName)`.dll in the WASM targets?

Reply:
The targets we are reusing here to build the wasm AppBundle are the ones that were added for our wasm tests. Those targets are hardcoding these two properties to point to our test output location, and to use the wasm test runner as the main assembly file name:
https://github.com/dotnet/runtime/blob/629ad508bf386fcbfd684a636d5e2e1146f84066/eng/testing/tests.mobile.targets#L202

What we are doing here is just to override those two settings so that we can reuse the rest of the logic on those targets to create the AppBundle